src/controllers/__tests__/pagoPAProxyController.test.ts

```diff
@@ -85,7 +85,9 @@ describe("PagoPAProxyController#getPaymentInfo", () => {
 
     const pagoPAClientFactory = new PagoPAClientFactory(
       process.env.PAGOPA_API_URL_PROD as string,
-      process.env.PAGOPA_API_URL_TEST as string
+      process.env.PAGOPA_API_KEY_PROD as string,
```

Comment:
can we extract this to avoid repeating the code?

Reply:
Sorry, I don't understand what you mean with avoid repeating code. Can you provide me an example?

Reply:
Yes, sorry, I pressed + and github commented just the line. I mean extracting this code outside the test cases:
```
const pagoPAClientFactory = new PagoPAClientFactory(
      process.env.PAGOPA_API_URL_PROD as string,
      process.env.PAGOPA_API_KEY_PROD as string,
      process.env.PAGOPA_API_URL_TEST as string,
      process.env.PAGOPA_API_KEY_UAT as string
    );
const pagoPAProxyService = new PagoPAProxyService(pagoPAClientFactory);
const controller = new PagoPAProxyController(pagoPAProxyService);
 ```

From what I see it seems it never changes.

Reply:
Fixed in [e5790f1](https://github.com/pagopa/io-backend/pull/874/commits/e5790f124d6f4beb72fbb0c64c847aa45db1ba4f)